sdk/python/feast/registry.py

```diff
@@ -717,6 +722,30 @@ def get_feature_view(
                 return FeatureView.from_proto(feature_view_proto)
         raise FeatureViewNotFoundException(name, project)
 
+    def get_stream_feature_view(
+        self, name: str, project: str, allow_cache: bool = False
+    ) -> StreamFeatureView:
+        """
+        Retrieves a stream feature view.
+
+        Args:
+            name: Name of stream feature view
+            project: Feast project that this stream feature view belongs to
+            allow_cache: Allow returning feature view from the cached registry
+
+        Returns:
+            Returns either the specified feature view, or raises an exception if
+            none is found
+        """
+        registry_proto = self._get_registry_proto(allow_cache=allow_cache)
+        for feature_view_proto in registry_proto.stream_feature_views:
```

Comment:
Is there a reason we moved this into a different field in the `Registry` proto? Could we have reused the existing `FeatureView` proto message?

Reply:
I don't think so, the proto for stream feature view and feature view are different.